server/server.go

```diff
@@ -201,7 +201,7 @@ func Run() error {
 	}
 
 	ch := make(chan os.Signal, 1)
-	signal.Notify(ch, syscall.SIGTERM, syscall.SIGINT, syscall.SIGKILL)
```

Comment:
Why did you remove `SIGKILL` from the list of signals to handle?

Reply:
because syscall.SIGKILL cannot be trapped. [http://linuxcommand.org/lc3_man_pages/kill1.html](http://linuxcommand.org/lc3_man_pages/kill1.html).